samples/net/wifi/apsta_mode/Kconfig

```diff
@@ -0,0 +1,49 @@
+# Copyright 2025 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+mainmenu "Network Wi-Fi apsta_mode application"
+
+config WIFI_AP_SSID
+	string "SSID of soft AP"
+	default "SOFT-AP"
```

Comment:
Please remove all the defaults here and below as these are related to the user configuration and are not generic settings that all users would use.